_reports/RaiVaibhav.md

```diff
@@ -0,0 +1,195 @@
+---
+categories:
+  - GSoC
+  - 2018
+  - StatusReport
+report_year: 2018
+date: 2018-08-12
+student: Vaibhav Kumar Rai
+organisation: coala
+organisation_link: https://coala.io
+project: Debug and Profile Bears
+project_link: https://summerofcode.withgoogle.com/projects/#6109762077327360
+tarball: https://rawgit.com/RaiVaibhav/GSoC-Report/master/project.tar.gz
+sha256sum: d61862f8f8be5ec168b07741c09781ef2dcc991e44d87edc05e6d237044052b0
+mentors: >
+  [Mischa Krüger](https://github.com/Makman2),
+  [Fabian Neuschmidt](https://github.com/fneu),
+  [Arjun Singh Yadav](https://github.com/arjunsinghy96)
+phase:
+ - Bonding: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/18
```

Comment:
https://github.com/coala/projects/pull/611#discussion_r209443765

the whole yaml needs to pass yamllint

see changes in the example